lib/msal-node/src/network/HttpClient.ts

```diff
@@ -176,14 +177,6 @@ const networkRequestViaProxy = <T>(
                     status: statusCode as number,
                 };
 
-                if ((statusCode < 200 || statusCode > 299) &&
-                    // do not destroy the request for the device code flow
-                    networkResponse.body["error"] !== Constants.AUTHORIZATION_PENDING) {
```

Comment:
This condition which relates to Device Code Flow seems to no longer be handled. With device code flow, we do polling until the user completes auth on a different device. I believe AAD returns 400 + authorization_pending error to show that user has not yet completed auth.